src/Components/Help.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+class Help extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      guessedCounty: "",
+    };
```

Comment:
I don't believe this is being referenced anywhere, If it's not you can remove it to clean up lines of code